dockstore-webservice/src/test/java/io/openapi/api/impl/ToolsImplCommonTest.java

```diff
@@ -0,0 +1,174 @@
+/*
+ *    Copyright 2017 OICR
```

Comment:
Nitpick: this is particularly out-of-date, the other one is 2019 :wink:

Reply:
fixed